tomls/omnibus-base-sepolia-andromeda/rewards/scbbtc_perps_liquidations_rewards.toml

```
@@ -0,0 +1,46 @@
+[provision.RewardsDistributor_scbbtc_liquidation_rewards]
+source = "synthetix-rewards-dist-ext:3.6.0"
+target = "synthetix-rewards-dist-ext:3.6.0@perps-scbbtc"
+options.salt = "<%= settings.salt %>"
```

```suggestion
options.salt = "<%= settings.salt %>-perps-scbbtc"
```
Also need to change salt =|